lib/selection/selection.dart

```diff
@@ -37,7 +37,7 @@ typedef E SelectionCallback<E>(datum, int index, Element element);
 typedef E SelectionValueAccessor<E>(datum, int index);
 
 /** Create a ChartedCallback that always returns [val] */
-SelectionCallback toCallback(val) => (datum, index, element) => val;
+SelectionCallback/*<T>*/ toCallback/*<T>*/(/*=T*/ val) => (datum, index, element) => val;
```

Comment:
do you want to make all the selection call backs (dynamic datum, int index, Element element)?